src/database/migrations/20241122100559-create-table-rollouts.js

```diff
@@ -0,0 +1,97 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('rollouts', {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER,
+			},
```

Comment:
@adithyadinesh0412 I'm thinking the frontend might use the same component for listing, which could make it challenging to send type and resource_type separately. Hence, I'm suggesting check with Akash. We can consider renaming resource_type to type, and to differentiate between program and solution rollouts, we can use rollout_type or target_type.